.github/workflows/publish.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish
+
+on:
+  push:
+    branches: [ main ]
+
+jobs:
+  build-and-deploy:
+    name: Build and deploy Gatsby site
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2.3.1
+
+      - name: Install Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      - name: Install Project Dependencies
+        run: yarn
+
+      - name: Install Gatsby CLI
+        run: npm install -g gatsby-cli@2.12.34
+
+      - name: Build
+        run: gatsby build
+
+      - name: Verify build
+        run: ls -la public
+
+      - name: Deploy with gh-pages
+        run: |
+          git remote set-url origin https://git:${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git
+          npm run deploy -- -u "github-actions-bot <support+actions@github.com>"
```

Comment:
`deploy` appears to also build, so we may be building twice. If we do it using npm or yarn we may not need to globally install the Gatsby CLI.

Reply:
If we use a [deploy action](https://github.com/BadIdeaFactory/reoyez/blob/100a327ad1faaedb12ab485ba7d8c9f264786d6c/.github/workflows/publish.yml#L29-L33), we may not need the deploy Node dep.

Reply:
good catch -- removed the build / verify build steps!

Reply:
Looks good. Could also get rid of `Install Gatsby CLI`.